discord/channel.py

```diff
@@ -39,6 +39,9 @@
     InviteTarget,
     SortOrder,
     StagePrivacyLevel,
+)
+from .enums import ThreadArchiveDuration as ThreadAutoArchiveDuration
```

Comment:
```suggestion
from .enums import ThreadArchiveDuration as ThreadArchiveDurationEnum
```
I think it's better if you import it as `ThreadArchiveDurationEnum`